lgc/patch/NggPrimShader.cpp

```diff
@@ -2855,25 +2875,34 @@ void NggPrimShader::exportPrimitiveWithGs(Value *startingVertexIndex) {
     break;
   }
   case OutputPrimitives::TriangleStrip: {
+    unsigned windingIndices[3];
+    if (m_pipelineState->getRasterizerState().provokingVertexMode == ProvokingVertexFirst) {
```

Comment:
Please also copy the explanative comments as the above.

Reply:
Done.